iib/workers/tasks/utils.py

```diff
@@ -766,10 +766,14 @@ def run_cmd(
                 raise IIBError(f'{exc_msg.rstrip(".")}: {match.groups()[0]}')
         elif cmd[0] == 'buildah':
             # Check for HTTP 50X errors on buildah
-            regex = r'.*(error creating build container).*((?:50[0-9]|125)\s.*$)'
-            match = _regex_reverse_search(regex, response)
-            if match:
-                raise ExternalServiceError(f'{exc_msg}: {": ".join(match.groups()).strip()}')
+            network_regexes = [
+                r'.*([e,E]rror:? creating build container).*(:?(50[0-9]|125)\s.*$)',
+                r'.*(read\/write on closed pipe.*$)',
+            ]
```

Comment:
Have you considered to get regular expressions from config file?
So one needn't deploy new version of IIB just to retry new network issue reported by buildah